### DISCUSSION TITLE
不偷着乐了，先Run为敬

### DISCUSSION THREAD

**Discussion by Chloee2000** [The-Run-Philosophy-Organization/run#416] [General]
以下是个人问题：
1.外语女生研究生转码学校推荐 
2.研究生和cs本科 时间+money成本
3.美国和加拿大比较

(陆本韩语，北美加勒比中文老师工作一年，已阅读站内所有润学文章) 感谢各位大神留言解答 T T

**Comment by skyabcher** [comment]
第一个问题回答不了
第二个问题，首选研究生。在美加的话，研究生两年时间，五十万可以读下来，本科生四年一百万吧。数字都是估算。
第三个问题，首选加拿大，拿身份要紧。
